app/src/models/vocabulary.model.js

```diff
@@ -24,6 +24,6 @@ const Vocabulary = new Schema({
     createdAt: { type: Date, default: Date.now },
     updatedAt: { type: Date, default: Date.now },
     status: { type: String, enum: STATUS, default: 'published' }
-});
+}, { usePushEach: true });
```

Comment:
what does this do? Can't seem to find any docs on it

Reply:
When I installed the service on my machine, no DB operation works in Mongoose - it keeps throwing an error "Unknown modifier $pushAll". 
I suspect this should be a problem specific of my environment, but I was not able to do any DB operations unless I had this in every model, and because of that I added this.

For reference, [here's where I found this](https://github.com/Automattic/mongoose/issues/5574#issuecomment-332290518). It might not be documented because I think this option was 3.5 specific and has been deprecated since.

I don't think that this will cause problems in production, but I am not sure! If you feel uncomfortable sending this to the master branch, let me know and I'll remove these changes from the PR :)

Reply:
update mongoose to the latest 5.x version and try again

Reply:
Yep, it fixed the problem, removed these lines in the models!